packages/webapp/pages/squads/create.tsx

```diff
@@ -137,18 +136,17 @@ function CreatePost(): ReactElement {
           onActiveChange={(active) => setDisplay(active)}
           controlledActive={display}
           shouldMountInactive={false}
-          className={{ header: 'px-1' }}
-          showHeader={!isNewMobileLayout || isTablet}
+          showHeader={isTablet}
```

Comment:
I think the condition here is not accurate.
```suggestion
          showHeader={!isMobile}
```

Reply:
It is correct with the current isTablet implementation, since isTablet stays true even in laptop sizes

Reply:
Yes, and with the condition of`! isNewMobileLayout`, this should be displayed on laptop.

Reply:
Oh, right. This is `true` on `isTablet`. Nevermind 🤦